src/main/kotlin/no/nav/familie/ba/mottak/hendelser/JournalføringHendelseAivenConsumer.kt

```diff
@@ -24,8 +24,8 @@ class JournalføringHendelseAivenConsumer(val journalhendelseService: Journalhen
     val journalføringshendelseAivenConsumerFeilCounter: Counter = Metrics.counter("joark.hendelse.journalføring.feil")
 
```

Comment:
Her må man ta kontakt med dokumentgjengen på #team_dokumentløsninger for å få lagt til familie-baks-mottak

Reply:
https://github.com/navikt/dokumenthandtering-iac/pull/59